metrics/network/iperf_dockerfile/Dockerfile

```diff
@@ -0,0 +1,16 @@
+# Copyright (c) 2018 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+
+# Usage: FROM [image name]
+FROM ubuntu:16:04
```

Comment:
I believe this should be `ubuntu:16.04` - that is s/:/./ - it did not build for me until I made that edit.